corehq/apps/app_manager/views/app_summary.py

```diff
@@ -175,6 +176,7 @@ class AppDataView(LoginAndDomainMixin, ApplicationViewMixin, View):
 
     urlname = 'app_data_json'
 
+    @method_decorator(require_can_edit_or_view_apps, name='dispatch')
     def get(self, request, *args, **kwargs):
```

Comment:
I didn't know you could use `method_decorator` on a method to decorate a different method. I assume you've verified that this works.